### ISSUE TITLE
Gene annotations track test broken

### ISSUE THREAD

**Issue by keller-mark** [higlass/higlass#877]
_Commit_: a426e08

#### Steps to reproduce
1. Run the gene annotations track tests or check this travis log error https://travis-ci.org/github/higlass/higlass/builds/661547809#L1163

#### Observed behavior
- `Script error. thrown` for the "changes the color of the minus strand" test.

#### Expected behavior
- Test succeeds / passes

**Comment by keller-mark** [comment]
Note that the test does pass sometimes so it probably involves a race condition or timing issue

**Comment by pkerpedjiev** [comment]
Just FYI, I may have fixed that in https://github.com/higlass/higlass/pull/874/

Currently working on another test failure.

**Comment by keller-mark** [comment]
Oh great!

**Comment by alexander-veit** [comment]
I don't think it is fixed. Just got it again. As Mark said, it fails only sometimes.
